numpy/lib/npyio.py

```diff
@@ -86,10 +86,14 @@ def __dir__(self):
         return object.__getattribute__(self, '_obj').keys()
 
 
-def zipfile_factory(*args, **kwargs):
+def zipfile_factory(file, *args, **kwargs):
+    """Create a zipfile.Zipfile. Allows for Zip64, as well as pathlib.Path
```

Comment:
Please use standard docstring. Summary line, blank line, extended summary. Here that would be something like
```
"""Create a zipfile.

Extended summary.
"""
```
If you are going to document it, might say more about the arguments.